typescript/sdk/src/deploy/verify/ContractVerifier.ts

```diff
@@ -112,6 +121,26 @@ export class ContractVerifier extends MultiGeneric<VerificationInput> {
     return result.result;
   }
 
+  private async isAlreadyVerified(
+    chain: ChainName,
+    input: ContractVerificationInput,
+  ) {
+    try {
+      const result = await this.submitForm(
+        chain,
+        ExplorerApiActions.GETSOURCECODE,
+        {
+          address: input.address,
```

Comment:
nit: should the `address` override be after the `...this.compilerOptions`? I doubt `this.compilerOptions` will ever have an `address` key but just in case